azavea/ism-watershed-wellness-snapshot
# Remove ipad address bar

## Overview
This isn't a native app, but we want it to operate as closely as possible to one. The address bar on iOS is very bright in comparison with the app, so removing it would be ideal.

Connects #73

### Demo
_I don't have my iPad on-hand, so I can't test this_ 

### Notes
- You can't "Add to homescreen" in the simulator so this has to be tested on a real iPad.
- I added custom favicons and changed the app name to "Watershed Wellness Snapshot" from "React App"
- The app will need to be opened from the homescreen for this to work.
- At the same time, I noticed that the intro screen's larger font was breaking into two lines—I've fixed that in this branch.

## Testing Instructions
* `git pull`
* Open app in Safari
* Tap the "share button", tap "Add to Homescreen"
* Close Safari
* Tap the new homescreen button for the app
* There should be no address bar 🤞
